pilot/lib/pilot/build_manager.rb

```diff
@@ -111,7 +114,8 @@ def should_update_build_information(options)
 
     # This method will takes care of checking for the processing builds every few seconds
     # @return [Build] The build that we just uploaded
-    def wait_for_processing_build(options)
+
+    def wait_for_processing_build(platform: nil)
```

Comment:
Doesn't `options` still need to be passed in, since it is used in this method body?